doc/guide_3rd_party.md

```diff
@@ -90,49 +90,50 @@ In order to start the grant consent user flow, redirect the user to the followin
-
-using var httpClient = new HttpClient();
-using var request = new HttpRequestMessage(HttpMethod.Get, $"https://demo.energytrackandtrace.dk/wallet-api/wallets?organizationId={organizationId}");
-request.Headers.Authorization = new AuthenticationHeaderValue("bearer", token);
-request.Headers.Add("X-API-Version", "20250101");
-var response = await httpClient.SendAsync(request);
-```
-
 Refer to OpenAPI spec for a description of available API endpoints.
 
-In the example above, an organization id is used to specify which organization to act on behalf of. The available organizations are given by the consents granted to you as a 3rd party client. A list of consenting organizations can be found with a `GET` request to `api/authorization/client/consents`, providing a valid access token that identifies you as a 3rd party client.
+Most API's takes an organization id as query param and is used to specify which organization to act on behalf of. The available organizations are given by the consents granted to you as a 3rd party client. A list of consenting organizations can be found with a `GET` request to `api/authorization/client/consents`, providing a valid access token that identifies you as a 3rd party client.
 
 ### C# example: Get consents
 
 ```csharp
 var token = "<access-token>"; // Access token obtained with client-credentials
 
-using var httpClient = new HttpClient();
-using var request = new HttpRequestMessage(HttpMethod.Get, "https://demo.energytrackandtrace.dk/api/authorization/client/consents");
+var httpClient = new HttpClient();
+var request = new HttpRequestMessage(HttpMethod.Get, "https://demo.energytrackandtrace.dk/api/authorization/client/consents");
 request.Headers.Authorization = new AuthenticationHeaderValue("bearer", token);
-request.Headers.Add("X-API-Version", "20230101");
+request.Headers.Add("X-API-Version", "1");
 var response = await httpClient.SendAsync(request);
+
+var consentResponse = await response.Content.ReadFromJsonAsync<ConsentResponse>();
 ```
 
 The result should look something lige the `JSON` response below. A list of organizations that have granted you consent to act on their behalf.
 
 ```json
 {
-  "result":[
-    { "organizationId":"645ca01a-7ddd-4d27-ba67-7abc550ce5e3", "organizationName":"Producent A/S"}]
+  "result":[{ "organizationId":"645ca01a-7ddd-4d27-ba67-7abc550ce5e3", "organizationName":"Producent A/S", "tin": "1234567" }]
 }
 ```
 
 Use the `organization id` for the corresponding organization when making request.
 
 ## Get Metering Points
 
-Obtain a list of metering points owned by an organization by using the <https://demo.energytrackandtrace.dk/developer#tag/MeteringPoints/paths/~1api~1measurements~1meteringpoints/get> endpoint.
+Given the Organizations you can now get metering points for one of them. Obtain a list of metering points owned by an organization by using the <https://demo.energytrackandtrace.dk/developer#tag/MeteringPoints/paths/~1api~1measurements~1meteringpoints/get> endpoint.
```

Comment:
"easily". This document is part of our repository. If it is important I think we should make a new task. But I would like it to point to demo, since you should not develop against prod :D

Reply:
easily. Skill issue ;)